### PR TITLE
Add follow me support for Breez Max

### PR DESCRIPTION
**Pull request by mikeyoon**
Adds support for the follow me feature. One thing I noticed that might need addressing is that I restricted it to the Breez Max. I don't know if this feature is limited to the Max, or if there's a better way to determine follow me compatibility altogether.

### CODE REVIEW

**Comment by miles267** [comment]
Thanks @mikeyoon. Am so excited to use this on my Breez Max. I’ve found the temp reading at the main air handler up high on wall to read differently than on wall where the BM thermostat is installed. How can we start using this new feature?

File: custom_components/cielo_home/cielohomedevice.py [hunk 1]
```diff
@@ -495,6 +519,7 @@ def _get_action(self) -> object:
             "fanspeed": self._device["latestAction"]["fanspeed"],
             "temp": self._device["latestAction"]["temp"],
             "swing": self._device["latestAction"]["swing"],
+            "followme": self._device["latestAction"]["followme"],
```
Comment (bodyscape) [inline custom_components/cielo_home/cielohomedevice.py:522]: Put this in a try exception. Not all device has this feature

**Review by bodyscape — CHANGES_REQUESTED** [review]

File: custom_components/cielo_home/switch.py [hunk 2]
```diff
@@ -23,13 +24,21 @@ async def async_setup_entry(
...
         )
+
         if device.get_is_appliance_is_freezepoin_display():
             entities.append(
                 CieloHomeSwitchFreezingPoint(
                     device, "Freezing Point", device.get_uniqueid() + "FreezingPoint"
                 )
             )
 
+        if device.get_device_type() == "BREEZ-MAX":
```
Comment (bodyscape) [inline custom_components/cielo_home/switch.py:35]: you must check if the device is support follow-me, not if it's Breeze max. My device support it and i don't have a Breeze-Max.

**Comment by audreez** [comment]
are you guys sure only max supports follow me ?

i have breez-plus and breez-eco and they both allow me to use follow me function which changes the temp on midea, can these be added?

**Comment by bodyscape** [comment]
Don't worry, i ask for he change that! It's not for only the breeze-max

**Pull request merged**